Wikipedia/Code/UIViewController+WMFChildViewController.swift

```diff
@@ -8,35 +8,6 @@ extension UIViewController {
     ///   - childController: Controller whose view will be embedded in containerView
     ///   - containerView: View to which childController's view will be added as a subview
     @objc public func wmf_add(childController: UIViewController!, andConstrainToEdgesOfContainerView containerView: UIView!) {
-        wmf_add(childController:childController, withContainerView: containerView, constraints: { make in
-            _ = make?.top.bottom().leading().and().trailing().equalTo()(containerView)
-        })
-    }
-
-    fileprivate func wmf_add(childController: UIViewController, withContainerView containerView: UIView, constraints: ((MASConstraintMaker?) -> Void)!){
-        guard
-            let childView = childController.view
-            else{
-                assertionFailure("Expected child controller view")
-                return
-        }
-        wmf_configureForDynamicHeight(childView: childView, containerView: containerView)
-        addChildViewController(childController)
-        containerView.addSubview(childView)
-        childController.view.mas_makeConstraints(constraints)
-        childController.didMove(toParentViewController: self)
```

Comment:
I think the `didMove` call is needed for container-ized vc's...

https://developer.apple.com/documentation/uikit/uiviewcontroller/1621405-didmove
`If you are implementing your own container view controller, it must call the didMove(toParentViewController:) method of the child view controller after the transition to the new controller is complete or, if there is no transition, immediately after calling the addChildViewController(_:) method.`

Reply:
Good catch, unintentionally removed that - re-added with latest commit